isChildTermOf

```diff
@@ -0,0 +1,63 @@
+  console.warn node_modules/ajv/lib/ajv.js:385
```

Comment:
i think this file got created by a test or something. it should be ok to delete it.